.github/maintainers_guide.md

```diff
@@ -40,11 +40,14 @@ Releases for this library are automatically generated off of git tags. Before cr
 To create a new release:
 
 1. Create a new GitHub Release from the [Releases page](https://github.com/slackapi/deno-slack-api/releases) by clicking the "Draft a new release" button.
-2. Input a new version manually into the "Choose a tag" input. Ensure that this version adheres to [semantic versioning][semver] based on what's being released. Version tags should match the following pattern: `1.0.1` (no `v` preceding the number).
-  - After you input the new version, click the "Create a new tag: x.x.x on publish" button.
+2. Input a new version manually into the "Choose a tag" input. You can start off by incrementing the version to reflect a patch. (i.e. 1.16.0 -> 1.16.1)
+    - After you input the new version, click the "Create a new tag: x.x.x on publish" button. This won't create your tag immediately.
+    - Auto-generate the release notes by clicking the "Auto-generate release notes" button. This will pull in changes that wiill be included in your release. 
+    - Review the pull request labels of the changes included in this release (i.e. `semver:minor` `semver:patch`, `semver:major`). Tip: Your release version should be based on the tag of the largest change, so if this release includes a `semver:minor`, the release version in your tag should be upgraded to reflect a minor. 
```

Comment:
Maybe we mention to flip to the Preview mode of the release notes so that one can hover over the PR numbers to get a snapshot view of the labels associated to the PR?